apps/frontend/src/assets/css/variables.css

```diff
@@ -3,17 +3,27 @@
 	--color-background-2: rgb(240 236 236);
 	--color-background-3: rgb(246 245 245);
 	--color-background-4: rgb(255 255 255);
+	--color-background-5: rgb(50 34 43);
+	--color-background-6: rgb(236 237 238);
+	--color-background-7: rgb(12 121 151);
```

Comment:
Let's have descriptive names, instead of numbering them. --color-background-7 says nothing to me about what color it might be